stl/inc/algorithm

```diff
@@ -4603,6 +4607,7 @@ namespace ranges {
         _STATIC_CALL_OPERATOR constexpr replace_copy_if_result<_It, _Out> operator()(
             _It _First, _Se _Last, _Out _Output, _Pr _Pred, const _Ty& _Newval, _Pj _Proj = {}) _CONST_CALL_OPERATOR {
             _STD _Adl_verify_range(_First, _Last);
+            _STD _Verify_ranges_do_not_overlap(_First, _Last, _Output);

```

Comment:
This is checking (first, last, output) but the (range, output) overload below isn't being checked.

Reply:
+lack of coverage, need to check every overload